src/flags.rs

```diff
@@ -247,32 +238,34 @@ fn test_v8_set_flags_preprocess_2() {
 
 // Pass the command line arguments to v8.
 // Returns a vector of command line arguments that v8 did not understand.
+#[cfg_attr(feature = "cargo-clippy", allow(stutter))]
 pub fn v8_set_flags(args: Vec<String>) -> Vec<String> {
   // deno_set_v8_flags(int* argc, char** argv) mutates argc and argv to remove
   // flags that v8 understands.
   // First parse core args, then convert to a vector of C strings.
-  let (argv, rest) = v8_set_flags_preprocess(args);
-  let mut argv = argv
-    .iter()
-    .map(|arg| CString::new(arg.as_str()).unwrap().into_bytes_with_nul())
-    .collect::<Vec<_>>();
+  let (args, rest) = v8_set_flags_preprocess(args);
 
   // Make a new array, that can be modified by V8::SetFlagsFromCommandLine(),
   // containing mutable raw pointers to the individual command line args.
-  let mut c_argv = argv
+  let mut raw_argv = args
+    .iter()
+    .map(|arg| CString::new(arg.as_str()).unwrap().into_bytes_with_nul())
+    .collect::<Vec<_>>();
+  let mut c_argv = raw_argv
     .iter_mut()
     .map(|arg| arg.as_mut_ptr() as *mut i8)
     .collect::<Vec<_>>();
-  // Store the length of the argv array in a local variable. We'll pass a
-  // pointer to this local variable to deno_set_v8_flags(), which then
+
+  // Store the length of the c_argv array in a local variable. We'll pass
+  // a pointer to this local variable to deno_set_v8_flags(), which then
   // updates its value.
-  let mut c_argc = c_argv.len() as c_int;
+  let mut c_argv_len = c_argv.len() as c_int;
```

Comment:
If you insist 🤷
I find "argc" perfectly readable

Reply:
The complaint was argc and argv being within a char. Pedantic!